### PR TITLE
Add M134 Minigun

### PR DESCRIPTION
**Pull request by Rail-Runner**
Add M134 Minigun item and vehicle part.
Also add it to item groups.

### CODE REVIEW

File: data/json/vehicle_parts.json [hunk 1]
```diff
@@ -3746,6 +3746,26 @@
...
+        "id" : "mounted_m134",
+        "name" : "mounted M134D-H Minigun",
+        "symbol" : "t",
+        "color" : "green",
+        "broken_symbol" : "#",
+        "broken_color" : "green",
+        "damage_modifier" : 80,
+        "durability" : 400,
+        "range" : 12,
+        "fuel_type" : "308",
```
Comment (BevapDin) [inline data/json/vehicle_parts.json]: This entry is not required fro turrets anymore. The ammo type is taken directly from the gun item.

Comment (Darkyhard) [inline data/json/vehicle_parts.json]: Got it.

File: data/json/items/ranged.json [hunk 2]
```diff
@@ -4194,7 +4194,38 @@
...
+    "color": "dark_gray",
+    "name": "M134D-H Minigun",
+    "description": "The M134 Minigun is a 7.62x51 mm NATO, six-barreled machine gun.  It features Gatling-style rotating barrels with an electric motor.  Its' recoil is too powerful for firing unmounted, but if you could attach it to a vehicle and find enough ammo for it, it would become a devastating weapon. ",
+    "price": 5500000,
+    "material": "hardsteel",
+    "flags": ["MODE_BURST", "NEVER_JAMS", "MOUNTED_GUN"],
+    "skill": "rifle",
+    "ups_charges": 1,
+    "ammo": "308",
+    "weight": 19051,
```
Comment (Rivet-the-Zombie) [inline data/json/items/ranged.json]: This weight is a bit light for even the base gun *without* the ammunition feed system and batteries; please add another six or eight kilos to the weight in order to account for them.

File: data/json/items/ranged.json [hunk 3]
```diff
@@ -4194,7 +4194,38 @@
...
+    "dispersion": 120,
+    "sight_dispersion": 200,
+    "aim_speed" : 0,
+    "recoil": 55,
+    "durability": 9,
+    "burst": 100,
+    "clip_size": 500,
+    "reload": 10000,
+    "loudness" : 150,
+    "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "rail", 1 ],[ "mechanism", 4 ],[ "sights", 1 ]]
```
Comment (kevingranade) [inline data/json/items/ranged.json]: Do these mods even make sense for a mounted gun?  For example, swapping the sight will not do anything.

Comment (Rivet-the-Zombie) [inline data/json/items/ranged.json]: Yeah, this really shouldn't have any mod slots at all.

**Comment by jokermatt999** [comment]
I think you might need to have a mod location list, even if it's empty to get it to merge properly, since git seems unhappy at the moment.

**Comment by BevapDin** [comment]
@jokermatt999 "valid_mod_location" is completely optional. Git is "unhappy" because the another PR changed ranged.json and git can't figure out how to the new content from this PR into the changed file. This situation is know as merge conflict.

**Comment by jokermatt999** [comment]
Ahh, that explains it. I couldn't view the details on the conflict. Thanks.

**Pull request merged**